core/templates/dev/head/pages/base.html

```diff
@@ -203,6 +204,9 @@ <h2>
 
     <script src="/templates/dev/head/components/CollectionCreationService.js"></script>
     <script src="/templates/dev/head/components/ExplorationCreationService.js"></script>
+    <script src="/templates/dev/head/components/TopicCreationService.js"></script>
+    <script src="/templates/dev/head/components/SkillCreationService.js"></script>
+    <script src="/templates/dev/head/components/StoryCreationService.js"></script>
```

Comment:
Why are you adding these three files here?

Reply:
I think I did mistake while resolving conflit! I have to look at it.